openfisca_us/variables/irs/outputs.py

```diff
@@ -875,11 +875,127 @@ def formula(tax_unit, period, parameters):
             return 0
 
 
+class is_retired_on_total_disability(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is retired on total disability"
+    documentation = "Whether this individual has retired on disability, and was permanently and totally disabled when they retired"
```

Comment:
To confirm, this can't be a combination of retired status and permanent/total disability status because the person had to have been disabled when they retired and not after?

If so, how about renaming to `retired_on_total_disability` to reflect the past status?